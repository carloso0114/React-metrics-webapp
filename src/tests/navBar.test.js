import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import NavBar from '../components/navBar/navBar';
import store from '../redux/configureStore';

it('renders correctly', () => {
  const tree = renderer
    .create(<Router><Provider store={store}><NavBar /></Provider></Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
