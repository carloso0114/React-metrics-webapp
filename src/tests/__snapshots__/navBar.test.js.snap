// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="sc-gsTEea eVUrAJ navbar"
>
  <a
    className="sc-dlfnuX habHiA"
    href="/"
    onClick={[Function]}
  >
     &lt; Home
  </a>
  <span
    className="sc-bdfBQB gqyrhZ"
  >
    COVID Analitycs Data
  </span>
  <span>
    <i
      className="bi bi-mic-fill h4 me-3"
    />
    <i
      className="bi bi-gear-fill h4"
    />
  </span>
</nav>
`;
